services/app-api/forms/cmit.ts

```diff
@@ -58,4 +58,43 @@ export const CMIT_LIST: CMIT[] = [
     dataSource: DataSource.Hybrid,
     options: "",
   },
+  {
+    cmit: 20,
+    name: "LTSS-6: Admission to a Facility from the Community",
+    uid: "20",
+    measureSteward: "CMS",
+    measureSpecification: [
+      MeasureSpecification.CMS,
+      MeasureSpecification.HEDIS,
+    ],
+    deliverySystem: [DeliverySystem.FFS, DeliverySystem.MLTSS],
+    dataSource: DataSource.Hybrid,
```

Comment:
Found out that LTSS-6 has an Administrative dataSource in this [file](https://docs.google.com/spreadsheets/d/1ztZ1DSYCUkKz9E_np5q-k9CgBoHYKEWUc_Fd2t3RJmc/edit?gid=1582150716#gid=1582150716)